extensions/wikia/TemplateClassification/services/UserTemplateClassificationService.class.php

```diff
@@ -35,12 +44,37 @@ class UserTemplateClassificationService extends TemplateClassificationService {
 	 * @throws \Swagger\Client\ApiException
 	 */
 	public function getType( $wikiId, $pageId ) {
-		$templateType = parent::getType( $wikiId, $pageId );
+		return $this->getMappedType( parent::getType( $wikiId, $pageId ) );
```

Comment:
What do you think about changing name to something like `getUserType` because not all types are mapped there